docs/sources/reference/components/otelcol/otelcol.exporter.syslog.md

```diff
@@ -0,0 +1,346 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.exporter.syslog/
+aliases:
+  - ../otelcol.exporter.syslog/ # /docs/alloy/latest/reference/components/otelcol.exporter.syslog/
```

Comment:
```suggestion
canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.exporter.syslog/
```

Not necessary on a new topic. We could add the current location, but the alias that was included here is for a location that this topic has never been published to.